### ISSUE TITLE
Display Vision Debug?

### ISSUE THREAD

**Issue by DanielArnett** [lasarobotics/FTCVision#53]
How do I view the CV debug screen as shown [here](https://raw.githubusercontent.com/lasarobotics/ftcvision/img/test2.png)? How is this enabled in ManualVisionSample?

Thanks for such an awesome library!

**Comment by smo-key** [comment]
@DanielArnett, thank you for the question!

The "debug screen" is packaged in the separate testing app, `ftc-cameratest`, in the root project directory. To run the demo:
1. Clone the Git repo (see readme) and make sure you are on the `master`or `dev` branches (stable releases go into `master` and development code in `dev`) by running `git checkout [branch]`. If you have not already done so, **follow the readme's directions**.
2. Copy the `ftc-cameratest` folder from the repo into your Android Studio project (see readme's migration instructions - just copy an additional folder)
3. Add the following line to your project's `settings.gradle`:

``` gradle
include `:ftc-cameratest`
```
1. Open the project in Android Studio. Click the green `Sync Project with Gradle Files` button in the navbar. Once sync is complete, click the dropdown in the navbar (between the green `Run` and `Build` icons) and select `ftc-cameratest`.
2. Run the project normally: this time, Android Studio will build and upload the FTC Camera Testing app.

As of the moment, the camera testing app's code is a bit messy, but here's what you need to know:
- The file to edit is located in `ftc-cameratest/src/java/com/lasarobotics/tests/camera/CameraTestActivity`.
- The `CameraTestActivity` contains a `onCameraFrame()`  method which contains the code identical to the `onCameraFrame()` method in the `ManualVisionOpMode` (and by extension `ManualVisionSample`). For now, test the code in the `CameraTestActivity` by copying the `onCameraFrame()` code from the sample to the code in the activity. Once you have edited the code to your satisfaction, you can test it in a robot controller by copying the code back into a `ManualVisionOpMode`.
- We're working on a way to "spoof" the robot controller, effectively running Vision-enabled OpModes from a testing app without having to do any copy and paste. If you wish to see the progress as of this moment (it's a bit buggy), checkout the `feature-integratedtester2` branch and run the tester.

I hope this helps.
Arthur

**Comment by DanielArnett** [comment]
Great! Thank you for the quick response and again for the awesome CV
library!

**Comment by smo-key** [comment]
@DanielArnett We just implemented a testing app that runs regular OpModes! Take a look at the `dev` branch and let us know how the `ftc-cameratest` app works. (You may see rapid screen flashes on the ZTE Speed - that is a known problem with the renderer on the ZTE.)

The `CameraTestActivity` extends a `VisionEnabledActivity` and calls `initializeVision()` to display the window. It then can run any `TestableOpMode` - see the one in the `ftc-cameratest` app for a sample.
